d2bs/kolbot/libs/OOG.js

```diff
@@ -975,15 +1041,33 @@ MainLoop:
 					do {
 						text = control.getText();
 
-						if (text instanceof Array && typeof text[1] === "string" && text[1].toLowerCase() === info.charName.toLowerCase()) {
-							control.click();
-							this.click(6, 627, 572, 128, 35);
+						if (text instanceof Array && typeof text[1] === "string") {
+							count++;
 
-							break;
+							if (text[1].toLowerCase() === info.charName.toLowerCase()) {
```

Comment:
should we be using .toLowerCase() in the other name checks?

Reply:
added lowercase check for findCharacter function